### DISCUSSION TITLE
Add support for clickable hyperlinks in notifications

### DISCUSSION THREAD

**Discussion by Smorty100** [ppy/osu#14536] [Ideas]
I am on linux and I get a bunch of errors every time I open it. Sadly I cant just click on the links in the notifications to open these links in a browser to troubleshoot. I have to open the log file manually and copy the link from there. It really is just an inconvenience, but it bugs people which are having a lot of errors and have to fix all of them.

**Comment by bdach** [comment]
can you actually copy the content of the "errors" here?

**Reply by Smorty100** [reply]
I just copied all of the text which looks like the errors which showed up ingame in the notifications in the top right corner. I will mention that by far not all of the errors in the txt file below showed up in the notifications. My feature request wasn't to fix my problems (which seem to be related to my graphic tablet) but about being able to click on links in the notification and lazer actually opening these links.
Here are the errors:
[the "errors".txt](https://github.com/ppy/osu/files/7070551/the.errors.txt)

**Reply by bdach** [reply]
the fact that this many errors are thrown in the first place is troubling, that should not be the case.

**Reply by Smorty100** [reply]
That is very true. I restarted Lazer after disabling tablet input and the no errors popped up. So my issue seems to be tablet specific. That's the reason I handle tablet input through then Linux tablet manager... So in my case, there is no point in fixing it. Tho, if needed, I can provide more i formation to make it easier for other users having the same issue.
